\section*{Discussion}

\subsection*{RecB dissociation does not depend on RecA loading}
One of the main questions asked in this study was how long RecBCD stays bound to DNA following DSB recognition. We have been able to estimate that in wild-type \ecoli, RecB stays bound to DNA for 10 to 15 seconds on average (Table \ref{tab:fit_results}), independently of the concentration of ciprofloxacin. Given RecBCD's high processivity (digesting several kilobases of DNA before recognising a Chi-site) and its processing speed ($\sim$1.6 kb/s\cite{Wiktor2018}), we can reasonably expect that most of RecB's binding time is explained by its DNA degradation prior to Chi recognition, followed by rapid disassembly after Chi. This led us to ask whether RecB dissociation from DNA was triggered by its interaction with RecA. Imaging RecB in a \dreca\ mutant confirmed that this is not the case, and that RecB dissociation occurs independently of RecA loading (Table \ref{tab:fit_mutants}). We can hypothesise that RecBCD's conformational change upon Chi recognition induces a destabilisation of the complex, thereby greatly reducing its processivity, and resulting in dissociation shortly after Chi. Interestingly, the \geneteneighty\ mutant had a reduced dissociation rate compared to wild-type cells, which highlights the importance of RecB's exonuclease activity in the process and confirms our previous hypothesis on DNA binding in this mutant\cite{Lepore2023}. It is possible that digestion of the unwound DNA strands is a pre-requisite for RecBCD dissociation, and that it is performed by other cellular exonucleases in the \geneteneighty\ mutant.

\subsection*{Using RecB as a marker to quantify DSB formation}
Given RecBCD's high affinity for DSBs \emph{in-vivo} and our ability to detect its binding to DNA, the system is a useful tool to detect the presence of DSBs in \ecoli. Our experiments allowed us to estimate the number of recruitment of RecB to DSBs per hour, under different ciprofloxacin concentrations (Figure \ref{Fig:recruitment}A), and in the \dreca\ and \geneteneighty\ mutants (Figure \ref{Fig:recruitment}B). In the wild-type, the general pathway for DSB repair suggests that RecBCD is recruited once per DSB (Figure \ref{Fig:pathways}). We can therefore reasonably expect that the observed number of RecB recruitments observed matches with the number of DSBs formed (1 to 8 per hour, depending on ciprofloxacin concentration). In the \dreca\ and \geneteneighty\ mutants however, the disruptions to the DSB repair pathway lead to multiple RecB recruitments per DSB. We are therefore not able to estimate the rate of DSB formation in these mutants.

It should be noted that ciprofloxacin is expected to produce double-sided DSBs, hence leading to two RecB recruitments per break. We did not take this into account in our estimation of the DSB formation rate, as it has been previously suggested that the two sides of a DSB are kept in close proximity during DNA repair\cite{Vickridge2017,Keyamura2019}, and our imaging setup would not allow us to separate two RecB spots located close together. Indeed, our timelapse images of RecB binding seldom show two spots next to each other, further supporting the idea that we are not able to resolve RecBCD binding to both sides of a DSB.

Using RecBCD as a probe to label free double-stranded DNA ends allows us to monitor DSB formation in live \ecoli\ in real time, with single-cell sensitivity. It does not require the DSBs to be site-specific, and does not disturb the repair pathway, which makes it possible to induce sub-lethal amounts of DSBs and to image the cell's response. This tool could be further improved in the future by substituting the Halo-tag with a bright and photostable fluorescent protein, which would enable long-term imaging, for example in a mother-machine microfluidic device.

\subsection*{Multiple RecB recruitments per DSB in the $\mathbf{\Delta}$\emph{reca} and RecB$\mathbf{_{1080}}$ mutants}

\begin{figure*}[htbp]
    \centering
    \includegraphics[width=\textwidth]{Figures/Fig_mutants_pathways.pdf}
    \caption{RecBCD recruitment pathways in wild-type \emph{E. coli}, \dreca\ and \geneteneighty\ mutants. \textbf{(Wild-type)} After DSB recognition, RecBCD degrades DNA until it recognises a Chi-site. It switches activity to create a 3' ssDNA overhang, and promotes RecA loading. The RecA-coated ssDNA can then be used for DNA repair by homologous recombination. \textbf{($\mathbf{\Delta}$reca)} In the absence of RecA, the 3' ssDNA is coated by SSB, and eventually digested by cellular nucleases. Blunting of the DNA end by digestion of the ssDNA creates a new substrate for binding of RecBCD. \textbf{(RecB$\mathbf{_{1080}}$)} Following DSB recognition, \teneighty\ unwinds DNA without digesting it. The unwound ssDNA can either be digested by nucleases, leading to a new blunt dsDNA end and RecBCD re-recruitment; or the RecFOR complex displaces SSB to load RecA, allowing DNA repair by homologous recombination to proceed.}
    \label{Fig:pathways}
\end{figure*}

Our observations of cell elongation (Supp. Figures \ref{SIFig:mutants_bf} and \ref{SIFig:mutants_cell_lengths}), RecB recruitment to DNA (Figure \ref{Fig:recruitment}), and nucleoid position (Figure \ref{Fig:nucleoid}) in the different mutant strains have led us to the model of RecB recruitment described in Figure \ref{Fig:pathways}, which reinforces and extends the model we previously proposed in \cite{Lepore2023}. In wild-type cells, a DSB is recognised by RecBCD, which promotes RecA loading. The RecA filament triggers the SOS response, and is used for homology search and repair. In the \dreca\ mutant, the 3' ssDNA generated by RecBCD is first coated by SSB, and then degraded by the SbcCD and ExoI nucleases\cite{Zahradka2009}. This leads to blunting of the DNA end, creating a new substrate on which RecBCD can bind. This circle leads to multiple RecBCD recruitments per DSB, and eventually to full chromosome degradation. In the \geneteneighty\ mutant, RecBCD unwinds DNA without degrading it. After the ssDNA is coated with SSB, two competing pathways take place: either DNA degradation by SbcCD and ExoI leading to DNA-end blunting and re-recruitment of RecBCD, or displacement of SSB by RecFOR and loading of RecA, leading to SOS induction and homologous repair.

\subsection*{Accumulation of RecA filaments under high DNA damage}
Under exposure to high ciprofloxacin concentrations (20-30 ng/ml), we observed an accumulation of cells that contained a RecA filament (Supp. Figures \ref{SIFig:reca_structures}A and \ref{SIFig:reca_structures}B). At these ciprofloxacin concentrations, we have determined that cells undergo frequent DSBs, leading to multiple recruitments of RecB to DNA over the course of the experiment (up to 8 per hour, Figure \ref{Fig:recruitment}A). Given RecBCD's high processivity\cite{Wiktor2018}, such a high number of RecB recruitments would likely result in several kilobases of DNA being digested at different chromosomal locations. Furthermore, it has been previously reported that DSB repair by RecBCD takes ~15 min from RecBCD binding to a DSB to completion of the repair\cite{Wiktor2021}. At a rate of 8 RecB recruitments per hour, it is likely that several DSBs would be processed simultaneously, resulting in fragmentation of the bacterial chromosome. If a homologous copy of the break site is not present in the cell, we expect that the repair process will stall at the homology search stage, after formation of the RecA filament, which is consistent with our observation that a large proportion of cells contain RecA filaments following exposure to over-MIC ciprofloxacin concentrations.

\subsection*{Centring of DNA-bound RecB molecules in the cell}
Imaging RecB simulteanously to the bacterial nucleoid reinforced our interpretation that long-lived RecB spots ($>$10 sec) are DNA-bound RecB molecules, and helped us understand their spatial distribution in the cell. Nucleoid compaction in \ecoli\ had previously been reported after UV irradiation\cite{Odsbu2014}, and we were able to confirm that a similar compaction occurs under ciprofloxacin treatment (Supp. Figure \ref{SIFig:nucleoid_compaction}). As expected, DNA-bound RecB molecules colocalised with the nucleoid. The close match between nucleoid density and DNA-bound RecB distribution hinted that RecB binding to DSBs occurred at random positions in the nucleoid, as would be expected given that ciprofloxacin causes DSBs at random sites in the genome.

Nucleoid imaging in the \dreca\ and \geneteneighty\ mutants pointed out the involvement of RecA loading (and probably SOS induction) in nucleoid compaction, again confirming previous observations\cite{Odsbu2014}. The absence of any significant nucleoid compaction in the \geneteneighty\ mutant despite its ability to induce the SOS response was surprising, and could result from a significant delay in SOS induction due to the loading of RecA by the RecFOR alternative pathway.
