\section*{Introduction}

% How DSBs get repaired
% RecBCD biochemistry (incl. dissociation from DNA)
DNA double-strand breaks (DSBs) pose a serious threat to the integrity of bacterial cells: when left unrepaired, they can lead to cell death, genome rearrangements or mutagenesis. In \emph{E. coli}, DSBs are repaired through the RecBCD pathway. RecBCD is a heterotrimeric protein complex, which combines several enzymatic activities\cite{Dillingham2008}. Upon recognition of a DSB, RecBCD translocates along the DNA at a speed of $\sim$1.6 kb/s\cite{Wiktor2018}, and digests both DNA strands until it meets a specific 8-base pair sequence (5'-GCTGGTGG-3') called Chi-site. Upon recognition of the Chi-site, RecBCD stops degrading the 3' end, leading to a 3' DNA overhang. Of note, Chi recognition by RecBCD is not systematic, and was previously reported to occur with $\sim$40\% probability\cite{Taylor1992,Cockram2015}. As RecBCD creates the 3' overhang, the nuclease domain of its RecB subunit promotes loading of the RecA protein onto the single-stranded DNA (ssDNA)\cite{Churchill2000, Spies2006}. The resulting RecA-DNA nucleoprotein filament is used as a template to search for an intact homologous sequence and perform homologous recombination. It is still unclear at which point exactly RecBCD dissociates from the DNA and what specifically triggers the dissociation.

% Endogenous DSBs
A common source of DSBs in \emph{E. coli} is the collision of the replication fork with DNA-bound proteins, which leads to the disassembly of the replisome\cite{Michel1997}. It has been reported that per cell cycle, $\sim$18\% of cells showed signs of endogenous DNA damage, most likely from replication fork breakage\cite{Sinha2018}. Replisome disassembly results in a Y-shaped DNA structure that is bound by the RuvAB complex, which pulls the DNA strands (in a process called "branch migration") to create a "chicken-foot" four-way DNA structure\cite{Seigneur1998}. The free end of this structure is a DNA double strand end, and is bound by RecBCD, which either (i) digests the whole DNA tail and displaces RuvAB, allowing the replisome to be re-loaded without RecA loading and SOS induction\cite{Seigneur1998}, or (ii) recognises a Chi-site and loads the RecA protein onto the 3' end of the DNA tail\cite{Michel2001}. Since this part of the DNA was recently replicated, a homologous sequence is likely to be present in close proximity, and can be used for homologous recombination.

% Exogenous DSBs (ciprofloxacin)
\emph{E. coli} cells might also experience DSBs from exo\-genous sources, such as anti\-biotics. Ciprofloxacin is an antibiotic of the fluoro\-quinolones family, which causes DSBs. In \emph{E. coli}, it does so by binding covalently to DNA topoisomerase II (gyrase), and trapping it in a DNA-bound conformation\cite{Kohanski2010}. The exact mechanism through which the DSBs are created is still unclear. Since the poisoned gyrase is tightly bound to DNA, it is likely to cause replication fork collisions\cite{Wentzell2000, Drlica2008}, and the subsequent repair of a chicken foot structure. Furthermore, the ciprofloxacin-poisoned gyrase is trapped in the "open" stage of its catalytic cycle, where it holds together two disjointed DNA ends. Therefore, upon removal of the gyrase (through a process that involves the ExoVII nuclease\cite{Huang2021}), it is possible that a DSB is formed independently of DNA replication\cite{Zhao2006}. Taken together, these studies suggest that ciprofloxacin causes both replication-dependent and independent DSBs to occur.

% General response to DNA damage (SOS and nucleoid compaction)
DSB repair is the first step of a general response to DNA damage known as the SOS response\cite{Baharoglu2014}. The formation of the RecA nucleoprotein filament triggers the auto-proteolysis of the LexA transcriptional repressor, which results in the activation of the genes of the SOS regulon. Among those are RecA, inhibitors of cell division such as SulA, and the SMC (structural maintenance of chromosomes)-like protein RecN. As a result, the cells elongate\cite{Bos2015}, and their DNA is compacted at the cell centre\cite{Odsbu2014}.

% RecB/RecA mutants
Mutations in proteins of the repair pathway have varying effects on the cell's ability to repair DSBs. Cells that are \emph{recA}-deficient (\dreca) are entirely unable to repair DSBs through homologous recombination, and the prolonged action of RecBCD on the DNA results in the digestion of large parts of the chromosome\cite{Horii1968, Chow2007}. \emph{recA}-deficient cells are however capable of processing DSBs arising from fork reversal, thanks to RecBCD's exonuclease activity\cite{Seigneur1998, Michel2001}. The $D_{1080} \rightarrow A$ point mutation in the RecB subunit of the RecBCD complex (\teneighty) abolishes RecB's nuclease activity, as well as its ability to load the RecA protein on DNA\cite{Yu1998, Wang2000}. The other activities of the complex (DSB recognition, DNA unwinding, Chi-recognition) are unaffected\cite{Anderson1999}. Despite the loss of its RecA loading activity, the \teneighty\ mutant is still able to repair DSBs. To do so, the \teneighty CD complex unwinds DNA at the break without degrading it, and the RecJ exonuclease degrades the 5' ssDNA. Finally, the RecOR complex displaces SSB and loads RecA, allowing for homologous recombination to occur\cite{Ivancic-Bace_2003}.

% Previous microscopy studies of DSB repair
Previous microscopy studies have shed light on different aspects of DSB repair by RecBCD. Several \emph{in-vitro} studies have quantified the rate of DNA unwinding by RecBCD\cite{Spies2003,Liu2013}, as well as the growth rate of RecA filaments\cite{Joo2006,Galletto2006,Handa2009} and the mechanism of homology searching\cite{Forget2012,Ragunathan2012}. Complimentary to the \emph{in-vitro} work, \emph{in-vivo} studies have provided insight into DNA end resection by RecBCD\cite{Wiktor2018}, confirming its translocation speed and high processivity. More recently, the diffusion of RecB was quantified \emph{in-vivo} at the single-molecule level\cite{Lepore2023}. The spatial organisation of RecA during DSB repair has been extensively characterised. Studies reported the presence of RecA foci\cite{Renzette2005,Renzette2007,Centore2007,Amarh2018}, filaments\cite{Kidane2005}, or bundles of filaments\cite{Lesterlin2013,Ghodke2019}. Of note, RecA imaging is particularly challenging because fluorescent protein fusions are not fully functional. The homology search and pairing processes were also observed \emph{in-vivo}, showing that DSBs are brought to the centre of the cell during the repair process\cite{Badrinarayanan2015,Wiktor2021}. Several aspects of DSB processing by RecBCD \emph{in-vivo} have not yet been fully elucidated: how long does RecBCD stay bound to DNA after recognising a DSB? Does the binding time change under different levels of DNA damage? Does RecA play a role in triggering RecBCD dissociation from DNA? These open questions are fundamental to the complete understanding of DSB repair in \emph{E. coli}, which would enhance future research into antimicrobial resistance.

% Overview of the article
To address these questions, we imaged RecB binding to DNA in living \emph{E. coli} cells exposed to various concentrations of ciprofloxacin. We measured the duration of DNA binding and found that it was not affected by the ciprofloxacin concentration within the tested range (0 to 30 ng/ml). Concurrently, imaging RecA and the bacterial nucleoid provided a more comprehensive view of the DNA repair process under our experimental conditions. We observed an accumulation of RecA filaments and compaction of the bacterial nucleoid, both of which scaled with ciprofloxacin concentration and duration of exposure. Furthermore, to investigate the role of RecA in the dissociation of RecB from DNA, we imaged RecB in the \dreca\ and \teneighty\ mutants, revealing that RecA does not influence RecB dissociation.
