\section*{Introduction}

% Endogenous DSBs
DNA double-strand breaks (DSBs) pose a serious threat to the integrity of bacterial cells. In \emph{E. coli}, a common source of DSBs is the collisions of the replication fork with DNA-bound proteins, which has been reported to occur in $\sim$18\% of cells per cell cycle\cite{Sinha2018} and leads to the disassembly of the replisome\cite{Michel1997}. The resulting Y-shaped DNA structure is bound by the RuvAB complex, which pulls the DNA strands (in a process called "branch migration") to create a "chicken-foot" four-way DNA structure\cite{Seigneur1998}. The free end of this structure (DNA tail) is bound by RecBCD, which either (i) digests the whole DNA tail and displaces RuvAB, allowing the replisome to be re-loaded, or (ii) recognises a Chi-site and loads the RecA protein onto the 3' end of the DNA tail\cite{Michel2001}. Since this part of the DNA was recently replicated, a homologous sequence is likely to be present in close proximity, and can be used for homologous recombination.

% Exogenous DSBs (ciprofloxacin)
\emph{E. coli} cells might also experience DSBs from exo\-genous sources, such as anti\-biotics. Ciprofloxacin is an antibiotic of the fluoro\-quinolones family, which causes DSBs. Im \emph{E. coli}, it does so by binding covalently to DNA topoisomerase II (gyrase), and trapping it in a DNA-bound conformation\cite{Kohanski2010}. The exact mechanism through which the DSBs are created is still unclear. Since the poisoned gyrase is tightly bound to DNA, it is likely to cause replication fork collisions\cite{Wentzell2000, Drlica2008}, and the subsequent repair of a chicken foot structure. Furthermore, the ciprofloxacin-poisoned gyrase is trapped in the "open" stage of its catalytic cycle, where it holds together two disjointed DNA ends. Therefore, upon removal of the gyrase (through a process that involves the ExoVII nuclease\cite{Huang2021}), we can reasonably expect the formation of a DSB independently of DNA replication\cite{Zhao2006}. Taken together, these studies suggest that in fast-growing cells, which contain multiple replication forks, both replication-dependent and independent DSBs occur.

% How DSBs get repaired
% RecBCD biochemistry (incl. dissociation from DNA)
In \emph{E. coli}, DSBs are repaired through the RecBCD pathway. RecBCD is a heterotrimeric protein complex, which combines several enzymatic activities\cite{Dillingham2008}. Upon recognition of a DSB, RecBCD translocates along the DNA at a speed of $\sim$1.6 kb/s, thanks to its combined 5' and 3' helicase activities\cite{Wiktor2018}. While translocating, RecBCD digests both DNA strands until it meets a specific 8-base pair sequence (5'-GCTGGTGG-3') called Chi-site. Upon recognition of the Chi-site, RecBCD pauses briefly before resuming translocation, albeit with modified nuclease activity: the 5' DNA end is be degraded more slowly, and the 3' end is not degraded at all, leading to a 3' DNA overhang. Of note, Chi recognition by RecBCD is not systematic, and was previously reported to occur with $\sim$40\% probability by an \emph{in-vitro studies}\cite{Taylor1992}. As RecBCD creates the 3' overhang, the nuclease domain of its RecB subunit promotes loading of the RecA protein onto the single-stranded DNA (ssDNA)\cite{Churchill2000, Spies2006}. The resulting RecA-DNA nucleoprotein filament is used as a template to search for an intact homologous sequence and perform homologous recombination. It is still unclear at which point exactly RecBCD dissociates from the DNA and what specificly triggers the dissociation. One \emph{in-vitro} study suggested that RecBCD dissociation was triggered by Chi recognition\cite{Taylor1999}, but the RecA filament could also play a role.

% General response to DNA damage (SOS and nucleoid compaction)
DSB repair is the first step of a general response to DNA damage known as the SOS response\cite{Baharoglu2014}. The formation of the RecA nucleoprotein filament triggers the auto-proteolysis of the LexA transcriptional repressor, which results in the activation of the genes of the SOS regulon. Among those are RecA, inhibitors of cell division such as SulA, and the SMC (structural maintenance of chromosomes)-like protein RecN. As a result, the cells elongate\cite{Bos2015}, and their DNA is compacted at the cell centre\cite{Odsbu2014}. This response facilitates homologous recombination between sister chromosomes, and ensures that cell division does not take place before paired chromosomes have been correctly separated.

% RecB/RecA mutants
Mutants in the repair pathway are affected in their ability to repair DSBs. Cells that are \emph{recA}-deficient (\dreca) are entirely unable to repair DSBs through homologous recombination, and the prolonged action of RecBCD on the DNA results in the digestion of large parts of the chromosome\cite{Horii1968, Chow2007}. \emph{recA}-deficient cells are however capable of repairing DSBs arising from fork reversal, thanks to RecBCD's exonuclease activity\cite{Seigneur1998, Michel2001}. The $D_{1080} \rightarrow A$ point mutation in the RecB subunit of the RecBCD complex (\teneighty) abolishes RecB's nuclease activity, as well as its ability to load the RecA protein on DNA\cite{Yu1998, Wang2000}. The other activities of the complex (DSB recognition, DNA unwinding, Chi-recognition) are unaffected\cite{Anderson1999}. Despite the loss of its RecA loading activity, the \teneighty\ mutant is still able to repair DSBs. \teneighty CD unwinds DNA at the break without degrading it, and the Single-Strand Binding protein (SSB) coats the 3' ssDNA, while the RecJ exonuclease degrades the 5' ssDNA. Finally, the RecFOR complex displaces SSB and load RecA, allowing for homologous recombination to occur\cite{Ivancic-Bace_2003}.

% Previous microscopy studies of DSB repair
Previous microscopy studies have shed light on different aspects of DSB repair by RecBCD. Several \emph{in-vitro} studies have quantified the rate of DNA unwinding by RecBCD\cite{Spies2003,Liu2013}, as well as the growth rate of RecA filaments\cite{Joo2006,Galletto2006,Handa2009} and the mechanism of homology searching\cite{Forget2012,Ragunathan2012}. These studies imaged single DNA strands during unwinding or RecA filament formation, sometimes with the help of FÃ¶rster Resonance Energy Transfer (FRET)\cite{Joo2006,Ragunathan2012}. Complimentary to the \emph{in-vitro} work, \emph{in-vivo} studies have provided more insight into DNA end resection by RecBCD\cite{Wiktor2018}, confirming the complexe's translocation speed and high processivity. More recently, the diffusion of RecB was quantified \emph{in-vivo} at the single-molecule level\cite{Lepore2023}. The spatial organisation of RecA during DSB repair has been extensively characterised. Studies reported the presence of RecA foci\cite{Renzette2005,Renzette2007,Centore2007,Amarh2018}, filaments\cite{Kidane2005}, or bundles of filaments\cite{Lesterlin2013,Ghodke2019}. The homology search and pairing processes were also observed \emph{in-vivo}, showing that DSBs are brought to the centre of the cell during the repair process\cite{Badrinarayanan2015,Wiktor2021}. Of note, RecA imaging is particularly challenging because fluorescent protein fusions are not functional (or only partially). To circumvent this problem, previous studies have used partially functional fusions to fluorescent proteins\cite{Kidane2005,Renzette2005,Renzette2007,Centore2007,Lesterlin2013,Klimova2020}, tandem fusions where the fluorescent fusion is expressed alongside the wild-type protein\cite{Amarh2018,Wiktor2021}, fluorescent fusions to a phage protein that specifically binds RecA filaments\cite{Ghodke2019} or a fusion to the ALFA tag\cite{Wiktor2021}.

% Overview of the article
In this study, we aimed to learn more about the repair of ciprofloxacin-induced DSBs by RecBCD and RecA. We took advantage of RecBCD's low copy number ($\sim$5 molecules per cell on average\cite{Lepore2019a}) to perform enhanced localisation microscopy\cite{Yu2006, Elf2007} using a fully functional fusion of RecB to the Halo-tag that was previously developed and tested in the lab\cite{Lepore2019a}. This allowed us to detect and localise DNA-bound RecB molecules \emph{in-vivo}, and to estimate how long RecBCD stays bound to DNA. By exposing cells to different ciprofloxacin concentrations during imaging, we were able to look at the evolution of the repair process over time, and assess how cells were coping with different levels of DNA damage. To broaden our view of the repair process, we imaged a fluorescent fusion to the RecA protein in the same conditions, using a tandem fusion of RecA to the SYFP2 fluorescent protein that was previously shown not to affect its function\cite{Wiktor2021}. Finally, imaging RecB in the \dreca\ and \teneighty\ mutants gave us insight into the factors that might affect RecBCD's dissociation from the DNA.
